### PR TITLE
added japanese specific styles for appearance and effects panels

### PR DESCRIPTION
**Pull request by DivyaPrabhakar**
References #2571 

Before: 

After:

### CODE REVIEW

**Comment by placegraphichere** [comment]
probably need to alter the `:before` on this too to line it up with the "+" button:

```:lang(ja) .dialog-effects-popover {
    width: 13rem;
    ul:before {
        left: 6.4rem;   
    }
}```

**Comment by iwehrman** [comment]
Reviewed by @placegraphichere. Back to @DivyaPrabhakar for fixes.

File: src/style/shared/dialog.less [hunk 1]
```diff
@@ -203,3 +204,14 @@ dialog::backdrop {
...
     -webkit-clip-path: polygon(0% 0%, 1.8rem 0%, 0% 1.8rem);
 }
+
+// LANGUAGE SPECIFIC STYLES //
+
+/**** JAPANESE *****/
+
+:lang(ja) .dialog-effects-popover {
+    width: 13rem;
+    ul:before {
```
Comment (placegraphichere) [inline src/style/shared/dialog.less]: new line nit, otherwise good

Comment (placegraphichere) [inline src/style/shared/dialog.less]: and the extra spaces

**Comment by placegraphichere** [comment]
yay

**Pull request merged**